README.md

```diff
@@ -9,8 +9,8 @@
 
 ## Features
 
-  - Supports multiple objects (tables) with multiple indicies (based upon boost::multi_index_container)
-  - State is persistant and shareable among multiple processes
+  - Supports multiple objects (tables) with multiple indices (based upon boost::multi_index_container)
+  - State is persistent and shareable among multiple processes
```

Comment:
I don't believe that is true for the current implementation.

Reply:
This is just correcting the spelling  though :-).

What is not true anymore? that the state is persistent? shareable?

Reply:
I believe we removed the shareable among multiple processes support.

Reply:
Is it this [change](https://github.com/AntelopeIO/chainbase/commit/b11c59f5fa0b6cf1fb0bed478a4e4e8a0a7a3afa)?

I think it just removes some checks, but if the processes implement synchronization, it should probably still work I think.

Reply:
I mean if all that means is that it can be run in shared-memory, then ok. I would have preferred to see this readme.md be rewritten with more useful information, but I'll not hold up the minor spelling corrections.

Reply:
Thanks Kevin!